packages/web/src/components/toast/mobile/Toast.tsx

```diff
@@ -1,13 +1,15 @@
 import { ReactNode } from 'react'
 
+import { Text, Flex, IconComponent } from '@audius/harmony'
 import cn from 'classnames'
 
-import styles from './Toast.module.css'
 import ToastLinkContent from './ToastLinkContent'
 
 // TODO: SK - Move this into Stems
 interface ToastProps {
   children?: JSX.Element
+  leftIcon?: IconComponent
+  rightIcon?: IconComponent
```

Comment:
i have come to prefer a premature optimization of just using `ReactNode` over `IconComponent`. recently any time i've used `IconComponent` for something design comes back with something in the future that needs to be a `ReactNode`...personal preference here though, this works

Reply:
not strongly opinionated, can change

Reply:
er ... it's annoying to do that because we have to typecast later anyway to IconComponent in order for the props to work. do you still prefer it then?

Reply:
oh yea the structure of passing in would change, would have to toss in the complete component. it's good here tho, may not change from needing an `IconComponent` in the future, so can keep as is